doc/source/merging.rst

```diff
@@ -78,34 +78,40 @@ some configurable handling of "what to do with the other axes":
 ::
 
     pd.concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
-              keys=None, levels=None, names=None, verify_integrity=False)
+              keys=None, levels=None, names=None, verify_integrity=False,
+              union_categoricals=False, copy=True)
```

Comment:
If I'm a new user seeing the docstring of `pd.concat`, I have no idea about `union_categoricals`. 

Or does `union_categoricals` really say for you what is happening here? In my mind what is happening  is `less_strict_categorical_checks` or so, but maybe you have a different picture in mind and I should change mine :-)